kb/plane_geometry/formal_statements/statements/statements_scs/stat_triangle_internal_angle_side_tan.scs

```diff
@@ -0,0 +1,14 @@
+stat_triangle_internal_angle_side_tan <- statement;;
+
+stat_triangle_internal_angle_side_tan -> rrel_key_sc_element: concept_triangle; nrel_internal_angle; nrel_side; nrel_tan;;
+
+stat_triangle_internal_angle_side_tan => nrel_main_idtf: [Акс.(треугольник;внутренний угол*;сторона*;тангенс*)] (*<- lang_ru;;*);;
+stat_triangle_internal_angle_side_tan => nrel_main_idtf: [Ax.(triangle;internal angle*;side*;tan*)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]